spec/unit/classes/server_spec.rb

```diff
@@ -157,4 +157,77 @@ class { 'postgresql::globals':
       is_expected.to contain_class('postgresql::repo').with_version('99.5')
     end
   end
+
+  describe 'create config entries' do
+    let(:params) {{ 'config_entries' => {'ensure' => 'present', 'name' => 'data_directory_spec', 'value' => '/var/pgsql'}}}
```

Comment:
The layout of this isn't quite right, it's missing a hash in the middle with the resource names. It should probably be:

    let(:params) {{ 'config_entries' => {'data_directory_spec' => {'ensure' => 'present', 'value' => '/var/pgsql'}}}}